RefrigeratorSaraFarkash/RefrigeratorSaraFarkash/CShelf.cs

```diff
@@ -0,0 +1,134 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Drawing;
+using System.Linq;
+using System.Security.Claims;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace RefrigeratorSaraFarkash
+{
+    class CShelf : IComparable<CShelf>
+    {
+
+        private static int countObj = 0;
+
+        private int id;
+
+        public int Id
+        {
+            get { return id; }
+        }
+
+        private int numberLevel;
+
+        public int NumberLevel
```

Comment:
if you need a positive integer you should use the correct type: uint
please see more C# types here: https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/builtin-types/built-in-types

Reply:
fixed